packages/components/src/components/others/SecurityCard/index.tsx

```diff
@@ -12,7 +12,6 @@ const StyledCard = styled(Card)`
     flex-direction: column;
     align-items: center;
     padding: 10px;
-    z-index: 9;
```

Comment:
I am not sure why these z-index in security cards were needed. If something gets broken I will fix it I promise! 😁